packages/react-components/react-toolbar/src/components/Toolbar/useToolbarStyles.ts

```diff
@@ -12,16 +12,28 @@ export const toolbarClassNames: SlotClassNames<ToolbarSlots> = {
 const useStyles = makeStyles({
   root: {
     display: 'flex',
+    ...shorthands.padding('4px', '8px', '4px', '8px'),
     ...shorthands.gap('8px'),
   },
+  vertical: {
+    display: 'flex',
```

Comment:
```suggestion
```

root already has this style, you don't need it again

Reply:
Done https://github.com/microsoft/fluentui/pull/24940/commits/71d16ef7bb67e7209204932c0280f661af2099b3